usr/sbin/p11sak/p11sak.c

```diff
@@ -1622,6 +1623,58 @@ static CK_RV pub_key_print_attributes(CK_SESSION_HANDLE session,
 
     return CKR_OK;
 }
+
+/**
+ * Alloc attribute
+ */
+static CK_RV tok_attribute_alloc(CK_SESSION_HANDLE session,
+                                 CK_OBJECT_HANDLE hkey,
+                                 CK_ATTRIBUTE_PTR attribute)
+{
+    CK_RV rv;
+    void *tmp;
+
+    if (!attribute)
+        return CKR_ARGUMENTS_BAD;
+
+    if ((attribute->pValue))
+        return CKR_CANCEL;
```

Comment:
Why double parentheses?

Reply:
Because ehhh... I have no idea. I'll change it.